### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by Diliaver**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,55 @@
...
+        else:
+            self.is_hungry = False
+            return 0
+
+
+class Cat(Animal):
+    def __init__(self, name, appetite=3, is_hungry=True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
```
Comment (DmytroSvirsa) [inline app/main.py:26]: You already use `super().__init__` from inherited Class `Animal` so it is code duplication:
```
self.name = name
self.appetite = appetite
self.is_hungry = is_hungry
```

File: app/main.py [hunk 2]
```diff
@@ -1 +1,55 @@
...
+    @staticmethod
+    def catch_mouse():
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(self, name, appetite=7, is_hungry=True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
```
Comment (DmytroSvirsa) [inline app/main.py:40]: You already use `super().__init__` from inherited Class `Animal` so it is code duplication:
```
self.name = name
self.appetite = appetite
self.is_hungry = is_hungry
```

File: app/main.py [hunk 3]
```diff
@@ -1 +1,55 @@
...
+    def bring_slippers():
+        print("The slippers delivered!")
+
+
+def feed_animals(list_of_animals):
+    sum_feed_points = 0
+    for i in list_of_animals:
+        if i.is_hungry is True:
+            sum_feed_points += i.appetite
+        if i.is_hungry is True:
```
Comment (DmytroSvirsa) [inline app/main.py:53]: the same check, unite

**Review by DmytroSvirsa — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (IvanRamyk) [reply app/main.py:26]: You haven't corrected it

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

**Review by IvanRamyk — APPROVED** [review]
Good job!

**Pull request closed without merging**